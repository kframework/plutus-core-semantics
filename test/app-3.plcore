(program

(module Foo (imported) (exported () ()) (local () ())
  (declare r2 integer)
  (define r2
    [(lam x (lam y (builtin addInteger x y))) 2 3 ]))

)
