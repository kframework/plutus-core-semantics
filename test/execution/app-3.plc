(program

(module Foo (import) (export () ())
  (declare r2 integer)
  (define r2
    [(lam x (lam y (builtin addInteger x y))) 2 3 ]))

)
