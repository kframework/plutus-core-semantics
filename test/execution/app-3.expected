<T>
  <k> . </k>
  <lenv> .Map </lenv>
  <genv> ( Foo . r2 ) |-> 5 </genv>
  <modules>
    <module>
      <curr> 0 </curr>
      <name> Foo </name>
      <data> .TyExpList </data>
      <decl> r2  .NameList </decl>
      <import> .ModNmList </import>
      <exportTy> .TyExpList </exportTy>
      <exportTm> .NameList </exportTm>
    </module>
  </modules>
</T>
